bundles/org.eclipse.text/src/org/eclipse/text/html/CoreHTMLPrinter.java

```diff
@@ -0,0 +1,465 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2018 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.text.html;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.net.URL;
+
+import org.eclipse.core.runtime.Platform;
+
+/**
+ * Provides a set of convenience methods for creating HTML pages.
+ * <p>
+ * Moved into this package from <code>org.eclipse.jface.internal.text.revisions</code>.</p>
+ *
+ * @since 3.14
+ */
+public class CoreHTMLPrinter {
```

Comment:
Can't we find a nicer name for this new API?

Reply:
> I'm naturally linguistically deficient in English, and to me, a printer is mostly 🖨️ and this class doesn't seem really related to 🖨️

I think this is related to printing on the screen or similar like in "System.out.**print**ln" also java has **Print**Writer class...

Reply:
Yes, I get it. Yet I'm not sure this class is really about printing on the screen, it's more about building HTML strings as far as I could see.